packages/core/test-helpers/src/websocket.ts

```diff
@@ -40,41 +44,78 @@ export const MockWsServer = Server
 export const mockWebSocketFlow = async (
   server: Server,
   flow: WsMessageExchange[],
+  options: {
+    enforceSequence: boolean
+    errorOnUnexpectedMessage: boolean
+  } = {
```

Comment:
Typescript can infer this type from the default
```suggestion
  options: {
```